classes/controllers/grid/feature/OrderItemsFeature.inc.php

```diff
@@ -21,13 +21,23 @@ class OrderItemsFeature extends GridFeature{
 	/** @var boolean */
 	var $_overrideRowTemplate;
 
+	/** @var string */
+	var $_nonOrderableItemMessage;
+
 	/**
 	 * Constructor.
+	 * @param $overrideRowTemplate boolean This feature uses row
+	 * actions and it will force the usage of the gridRow.tpl.
+	 * If you want to use a different grid row template file, set this flag to
+	 * false and make sure to use a template file that adds row actions.
+	 * @param $nonOrderableItemMessage string A translated message to be used
+	 * when user tries to move a non orderable grid item.
 	 */
-	function OrderItemsFeature($overrideRowTemplate) {
+	function OrderItemsFeature($overrideRowTemplate, $nonOrderableItemMessage) {
```

Comment:
Should this perhaps have a default for `$nonOrderableItemsMessage`? Most grids using `OrderItemsFeature` won't have non-orderable items, I would expect, so shouldn't be forced to specify this.